src/containers/locations-list/index.tsx

```diff
@@ -77,15 +77,15 @@ const LocationsList = ({ onSelectLocation }: { onSelectLocation?: () => void })
     return (
       <CellMeasurer key={key} parent={parent} cache={cache} columnIndex={0} rowIndex={index}>
         {({ registerChild }) => (
-          <div style={style} ref={registerChild}>
+          <div style={style} ref={registerChild} className="no-scrollbar md:px-6">
```

Comment:
![Screenshot 2023-06-14 at 08 35 54](https://github.com/Vizzuality/mangrove-atlas/assets/33252015/6dfc33ca-6896-4d7d-8573-5e47d7a0da64)
please, review paddings

https://www.figma.com/file/GJCx8yGfx6JERPuxlpG3nK/Mangrove-Watch-%5BInternal%5D?type=design&node-id=0-358&t=e5OeC5PBmOs7zSqx-0

Reply:
i am not able to identify the bug. 
![Screenshot 2023-06-14 at 09 50 40](https://github.com/Vizzuality/mangrove-atlas/assets/51995866/55804495-b744-435e-8148-a78d48b5edee)

Reply:
Me neither.